build-with-docker.sh

```diff
@@ -0,0 +1,2 @@
+docker run -it --rm --name dynamodb-cross-region-library -v "$PWD":/usr/src/mymaven -w /usr/src/mymaven maven:3.2-jdk-7 mvn clean install
+docker build -t vungle/dynamodb-cross-region-replication:1.1.0 .
```

Comment:
Do not specify your own Docker hub namespace here. Tag should be passed in as a parameter